core/test/Time_test.cpp

```diff
@@ -202,6 +202,27 @@ TEST_CASE("Durations", "[Duration]")
 
     Duration maybeADay = tt_day - tt;
     REQUIRE(maybeADay.seconds() == 1 * days);
+
+    Duration tenSeconds(10.);
+    REQUIRE(tenSeconds.seconds() == 10);
+
+    Duration hundredSeconds(100.);
+    REQUIRE(hundredSeconds.seconds() == 100);
+
+    Duration thousandSeconds(1000.);
```

Comment:
Comment: testTimeClasses doesn't fit the naming convention for text executables. I see why the choice was made (to avoid that one letter difference) but.....

Reply:
Convention is a strong word, but I belatedly agree.